configs/icnet/icnet_r101-d8_832x832_160k_cityscapes.py

```diff
@@ -0,0 +1,7 @@
+_base_ = [
```

Comment:
Can derive the configs of r101 and r18 from r50.
E.g. this config can derive from `configs/icnet/icnet_r50-d8_832x832_160k_cityscapes.py`